meshtastic/mesh_interface.py

```diff
@@ -315,6 +316,10 @@ def sendData(
             onResponse -- A closure of the form funct(packet), that will be
                     called when a response packet arrives (or the transaction
                     is NAKed due to non receipt)
+            onResponseAckPermitted -- should the onResponse callback be called
+                    for regular ACKs (True) or just data responses & NAKs (False)
+                    Note that if the onResponse callback is called 'onAckNak' this
+                    will implicitly be true.
```

Comment:
Right now this is only added to sendData, but I will try to go through and add it to other helpers too, for library users' sake.